vscode_extension/src/providers/AacDefinitionsViewProvider.ts

```diff
@@ -54,26 +57,59 @@ export class Definition extends vscode.TreeItem {
     private definitionType: string;
 }
 
-function getDefinitionByName(definitionName: string) {
-    return Promise.resolve(aacRestApi.getDefinitionByNameDefinitionGet(definitionName, true));
+export async function createDefinition() {
+    // Prompt the user to pick an AaC file to store the new definition in.
+    const definitionFileOpenDialogOptions: vscode.OpenDialogOptions = {
+        title: "Select the AaC file for the new definition.",
+        canSelectMany: false,
+        canSelectFolders: false,
+    };
+
+    let fileUris: vscode.Uri[] | undefined = await vscode.window.showOpenDialog(definitionFileOpenDialogOptions);
+
+    // Prompt the user to name the new definition.
+    const newDefinitionNameInputBoxOptions: vscode.InputBoxOptions = {
```

Comment:
Would this not be part of the definition content in the form?

Reply:
Yes it is, and I wasn't able to get the name field to prepopulate with the dialog choice. We're using the value to give the editor window a unique name, so we'll need an alternative to using the definition name as a unique file identifier.